backend/app/services/auth_service.py

```
@@ -125,35 +126,40 @@ def verify_google_token(request: GoogleLogin) -> GoogleUserData:
+        )
+
+        #  Exchange authorization code for credentials
+        flow.fetch_token(code=request.code)
+        credentials = flow.credentials
+
+        decoded_token = id_token.verify_oauth2_token(
+            credentials.id_token,
+            google_requests.Request(),
+            settings.GOOGLE_CLIENT_ID,
+        )
+    except Exception as e:
```

Ruff (0.12.2)

Do not catch blind exception: `Exception`

(BLE001)